content/en/docs/v3.1/benchmarks/etcd-3-demo-benchmarks.md

```diff
@@ -43,4 +43,4 @@ The performance is nearly the same as the one with empty server handler.
 The performance with empty server handler is not affected by one put. So the
 performance downgrade should be caused by storage package.
 
-[etcd-v3-benchmark]: ../op-guide/performance/#benchmarks
+[etcd-v3-benchmark]: /docs/v3.1/op-guide/performance/#benchmarks
```

Comment:
Can you leave this as a relative link -- here and in the other pages where refs were changed to start with `/docs/v3.1`?